### PR TITLE
Fix multiple error messages

### PR DESCRIPTION
**Pull request by 4p00rv**
Fix for this [trello card](https://trello.com/c/Uj6yisX7/109-different-error-messages)

### CODE REVIEW

File: src/templates/toolkit/bet/static.html.tt [hunk 1]
```diff
@@ -50,7 +50,7 @@
                         </div>
                         <div class="big-col">
                            <div id="expiry_type_duration">
-                              <input id="duration_amount" type="number" min="1" max="366" class="small_width_input" /><label></label>
```
Comment (raunakkathuria) [inline src/templates/toolkit/bet/static.html.tt]: @4p00rv why we remove `min, max`, we use to validate the form. @negar-binary can you confirm this

**Comment by raunakkathuria** [comment]
@4p00rv Can you add description of changes done and why it is required?

File: src/javascript/binary/pages/trade/event.js [hunk 2]
```diff
@@ -237,8 +237,7 @@ var TradingEvents = (function () {
         var amountElement = document.getElementById('amount');
         if (amountElement) {
             amountElement.addEventListener('input', debounce( function(e) {
-                var amount_val = parseFloat(e.target.value);
-                if (e.target.value % 1 !== 0 && ((+amount_val).toFixed(10)).replace(/^-?\d*\.?|0+$/g, '').length>2) {
+                if (e.target.value % 1 !== 0 && isStandardFloat(parseFloat(e.target.value))) {
```
Comment (raunakkathuria) [inline src/javascript/binary/pages/trade/event.js]: as you are checking this `e.target.value % 1 !== 0` everywhere why not move it to `isStandardFloat`

File: src/templates/toolkit/bet/static.html.tt (hunk 1 above)
Comment (4p00rv) [inline src/templates/toolkit/bet/static.html.tt]: As discussed with Ashkan, we are only showing one error everywhere not multiple errors. The backend already replies with errors, if any, and is displayed in the frontend!

**Pull request merged**